fedimint-server/src/consensus/aleph_bft/data_provider.rs

```diff
@@ -100,6 +108,11 @@ impl aleph_bft::DataProvider<UnitData> for DataProvider {
 
         assert!(bytes.len() <= ALEPH_BFT_UNIT_BYTE_LIMIT);
 
+        self.timestamp_sender
+            .send((Instant::now(), bytes.len()))
```

Comment:
Lets use a consensus hash here.

Reply:
To be clear: send the hash instead of the length over the channel? Sounds good to me (or if we are stingy with bytes: take the first 8 bytes of the hash, should still be plenty collision-resistant and keeps the same size as length).

Reply:
Having to clone here `bytes.clone()` is where the performance cost is. Otherwise it doesn't matter.

I can probably do `len ^ bytes[..8]` kind of thing, which is cheap and use both len and encoding itself.